cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/JobInfo.java

```diff
@@ -70,6 +70,8 @@ default String getId()
 
     boolean skipExtendedVerify();
 
+    boolean skipRowsViolatingConstraints();
```

Comment:
We should add javadocs in all public interfaces. This project lacks a log of javadocs in many places, but I would like new contributions to start adding better documentation